Example/Source/Examples/Collections/SimpleCollectionBrickViewController.swift

```diff
@@ -54,9 +54,9 @@ class SimpleCollectionBrickViewController: BrickApp.BaseBrickController {
 
 extension SimpleCollectionBrickViewController: CollectionBrickCellDataSource {
 
-    func configureCollectionBrickViewForBrickCollectionCell(brickCollectionCell: CollectionBrickCell) {
-        brickCollectionCell.brickCollectionView.registerBrickClass(LabelBrick.self)
-        brickCollectionCell.brickCollectionView.registerBrickClass(ImageBrick.self)
+    func registerBricks(for cell: CollectionBrickCell) {
+        cell.brickCollectionView.registerBrickClass(LabelBrick.self)
```

Comment:
As you are registering just the class, shouldn't you add these brickTypes in the initializer of the CollectionBrick?